hls4ml/model/quantizers.py

```diff
@@ -76,6 +90,10 @@ def __call__(self, data):
         ones = np.ones_like(data)
         return np.where(data > 0.5, ones, np.where(data <= -0.5, -ones, zeros))
 
+    def serialize_state(self):
+        state = {}
```

Comment:
Minor, but not sure if I am missing something obvious here: the BinaryQuantizer has a state but the TernaryQuantizer doesn't? In my understanding of the serialization, neither should have the state? Or if yes, both?

Reply:
This is a weird one... The "state" here is simply the choice of the number of bits used (1 or 2). For ternary we will always use 2 bits so there's no state to save. State is something you will need to pass to `__init__(...)` to recreate the object.